app/lib/en.coffee

```diff
@@ -1132,8 +1132,9 @@ module.exports =
+    p8: '''The relaunched site also allowed us more flexibility to include smaller sets of galaxies. We have included images taken with the United Kingdom Infrared Telescope (UKIRT), in Hawaii, for the recently completed <a href="http://www.ukidss.org/">UKIDSS</a> project. UKIDSS is the largest, deepest survey of the sky at infrared wavelengths. The UKIDSS infrared-light pictures and the SDSS visible-light pictures are different for two reasons. Firstly, older stars are brighter in infrared wavelengths (while in visible-light we see younger stars), and secondly, infrared light, like a fog light, can penetrate the dust and smoke in the centres of galaxies. Examining the UKIDSS images in Galaxy Zoo will enable us to see how old and young stars are arranged differently in the galaxies, and will reveal more of the structure in the inner parts of galaxies.'''
+    p9: '''The data being put into Galaxy Zoo can now change quickly, thanks to the dedicated work of volunteers and the &quot;retirement&quot; of images that have already been completed. One of the new sets of images to be classified comes from the Dark Energy Camera Legacy Survey (DECaLS), a survey which uses DECam, one of the most sensitive and widest-area cameras ever built and a key tool in our search to measure and understand <a href="https://en.wikipedia.org/wiki/Dark_energy">dark energy</a>. The camera, which is mounted on the 4-m Blanco telescope at CTIO in Chile, images the sky at similar wavelengths to SDSS, but with better resolution and with the ability to detect much fainter galaxies. These images are extending the reach of the science we began with Galaxy Zoo 2 and SDSS. Recently, the site also had Galaxy Zoo's first-ever images of fully simulated galaxies; these were images that were created entirely on a computer, using sophisticated simulations of the Universe starting just after the Big Bang and then watching as dark matter, gas, and stars evolve to form the galaxies we see today. The classifications of these images, which came from the <a href="http://www.illustris-project.org/">Illustris</a> project, are a vital test of the physics that go into these simulations.'''
+    p10: '''The images on the site today include our second set of DECaLS images, this time taken from the survey's second data release. This new data release covers a larger protion of the sky than the first. Finally, we have included a set of &quot;lost images&quot; from SDSS that should have been included in the Galaxy Zoo 2 sample. These images will fill in the hole that currently exists in the bright galaxy data set.'''
```

Comment:
"portion" is misspelled. 

Also - might be useful to add a date? Something like "starting in March 2016".